modules/jdbc/src/test/java/org/testcontainers/containers/JdbcDatabaseContainerTest.java

```diff
@@ -0,0 +1,71 @@
+package org.testcontainers.containers;
+
+import java.sql.Connection;
+import java.sql.SQLException;
+import lombok.NonNull;
+import org.junit.Test;
+import org.slf4j.Logger;
+
+import static org.assertj.core.api.Assertions.assertThatExceptionOfType;
+import static org.mockito.Mockito.mock;
+
+public class JdbcDatabaseContainerTest {
+
+    @Test
+    public void tanExceptionIsThrownIfJdbcIsNotAvailable() {
```

Comment:
I wonder what exactly `tan` prefix means ? :)

Reply:
I suppose this is a typo and it was supposed to mean `anExceptionIsThrownIfJdbcIsNotAvailable` 🙂